src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -173,7 +175,7 @@ export default class TelemetryTable extends EventEmitter {
     this.removeTelemetryCollection(keyString);
 
     let sortOptions = this.configuration.getConfiguration().sortOptions;
-    requestOptions.order = sortOptions?.direction ?? ORDER.DESCENDING; // default to descending
+    requestOptions.order = sortOptions?.direction ?? this.sortOptions.direction; // default to descending
```

Comment:
the comment on this line doens't seem relevant anymore

Reply:
truth bomb. done!